api.planx.uk/modules/analytics/metabase/collection/types.ts

```diff
@@ -0,0 +1,55 @@
+import { z } from "zod";
+import type { ValidatedRequestHandler } from "../../../../shared/middleware/validate.js";
+
+type ApiResponse<T> = {
+  data?: T;
+  error?: string;
+};
+
+/** Interface for incoming request, in camelCase */
+export interface NewCollectionParams {
+  name: string;
+  description?: string;
+  /** Optional; if the collection is a child of a parent, specify parent ID here */
+  parentId?: number;
+}
+
+/** Interface for request after transforming to snake case (Metabase takes snake while PlanX API takes camel) */
+export interface MetabaseCollectionParams {
+  name: string;
+  description?: string;
+  parent_id?: number;
+}
+
+/** Metbase collection ID for the the "Council" collection **/
+// const COUNCILS_COLLECTION_ID = 58;
```

Comment:
Yep, that's why it's commented out. Is there a better way to handle this in the meantime?

Reply:
I think there's three main approaches - 
 - We set up `COUNCILS_COLLECTION_ID` as an env var and pass in a valid ID for staging/pizza/localhost - this will differ per dev so not ideal but perfectly workable as an initial step
 - We commit to Metabase being prod only, and add a middleware to all `/metabase` routes on our API which just exits early if we're not on production - makes testing tricky!
 - We set up Metabase on staging and sync data down
 
We might end up on the third option, but let's work our way there via 1 and 2 first!